src/main/java/org/breedinginsight/model/SubscribedProgram.java

```diff
@@ -0,0 +1,17 @@
+package org.breedinginsight.model;
+
+import lombok.*;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class SubscribedProgram {
```

Comment:
Can this change to `SubscribedOntology`?

Reply:
changed